api/src/lib/db/client.js

```diff
@@ -0,0 +1,25 @@
+import pg from 'pg';
+import coPg from 'co-pg';
+import named from 'node-postgres-named';
+
+export default function* dbClient({host, port, user, password, database}) {
+    const postgresql = coPg(pg);
+    const connection = `postgres://${user}:${password}@${host}:${port}/${database}`;
+    const connectionResponse = yield postgresql.connect_(connection);
+    const client = connectionResponse[0];
+    const done = connectionResponse[1];
```

Comment:
I always have this error, I tested many fixes, babel transformer, runtime, polyfill ... But I failed everytime.

```/home/kmaschta/Projects/Marmelab/javascript-boilerplate/api/src/lib/db/client.js:30
var _marked = [dbClient].map(_regeneratorRuntime.mark);
                             ^

ReferenceError: _regeneratorRuntime is not defined
    at Object.<anonymous> (client.js:5:34)
    at Module._compile (module.js:435:26)
    at loader (/home/kmaschta/Projects/Marmelab/javascript-boilerplate/node_modules/babel-register/lib/node.js:130:5)
    at Object.require.extensions.(anonymous function) [as .js] (/home/kmaschta/Projects/Marmelab/javascript-boilerplate/node_modules/babel-register/lib/node.js:140:7)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/home/kmaschta/Projects/Marmelab/javascript-boilerplate/api/src/server.js:43:15)
    at Module._compile (module.js:435:26```